views/showcase.html

```diff
@@ -176,7 +176,7 @@ <h2 class="bg-gray-200 px-4 py-3 font-bold mt-4 mb-1">{{__('Metadata')}}</h2>
             {% for group in dataset.groups %}
             <p
               <span class="colGroup break-words">
-                {{group.title}}
+                <a href="/search?q=groups:{{group.name | safe}} {{ query.q }}" class="text-secondary">{{group.title}}</a>
```

Comment:
@gavram it is easy to simply link to group page, e.g., `/collections/{{ group.name }}`

Reply:
I fixed this as you suggested.